zephyr/Kconfig

```diff
@@ -40,6 +40,7 @@ config ZEPHYR_DP_SCHEDULER
 	default n
 	depends on IPC_MAJOR_4
 	depends on ZEPHYR_SOF_MODULE
+	depends on ACE
```

Comment:
The wording is a bit odd in the sense this is generic code that should work with any SOF platform, but the validated part makes sense (as does the change itself).